webindex/modules/core/src/main/java/org/apache/fluo/webindex/core/Constants.java

```diff
@@ -1,18 +1,19 @@
 /*
- * Copyright 2015 Webindex authors (see AUTHORS)
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- * 
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
```

Comment:
Some of these license headers seem to have inconsistent wrapping. I wonder if we can fix that with smart use of license-maven-plugin in future versions of the fluo-parent POM.